### DISCUSSION TITLE
searching for a long time url for tv malta

### DISCUSSION THREAD

**Discussion by ARAB-IPTV** [iptv-org/community#579] [Q&A]
im searching for a long time url for the maltian tv
TVM 
https://tvmnews.mt/tvm-live/
TVM NEWS 
https://tvmnews.mt/tvm-news-plus-live/
NET TV 
https://netondemand.mt/

**Comment by Sphinxroot** [comment]
https://smashmalta.com/smash-iptv

#EXTINF:-1,Smash TV
http://a4.smashmalta.com:82/live/smash/smash/5.ts
#EXTINF:-1,Smash 7News
http://a3.smashmalta.com:82/live/smash/smash/29.ts
#EXTINF:-1,Smash TV Shopping
http://a3.smashmalta.com:82/live/smash/smash/14.ts
#EXTINF:-1,Smash Food
http://a3.smashmalta.com:82/live/smash/smash/15.ts
#EXTINF:-1,Smash Politika
http://a3.smashmalta.com:82/live/smash/smash/19.ts
#EXTINF:-1,Smash Explore
http://a6.smashmalta.com:82/live/urusei/yatsura/3.ts
#EXTINF:-1,Smash Movies
http://a3.smashmalta.com:82/live/smash/smash/27.ts
#EXTINF:-1,Smash Jazz
http://a3.smashmalta.com:82/live/smash/smash/10.ts
#EXTINF:-1,Smash Music
http://a3.smashmalta.com:82/live/smash/smash/8.ts
#EXTINF:-1,Smash Country
http://a6.smashmalta.com:82/live/urusei/yatsura/1.ts
#EXTINF:-1,Smash House of Rock
http://a6.smashmalta.com:82/live/urusei/yatsura/2.ts
#EXTINF:-1,Smash Radio 104.6FM
http://a3.smashmalta.com:82/live/smash/smash/21.ts
#EXTINF:-1,Smash Jazz Notes (Radio)
http://a3.smashmalta.com:82/live/smash/smash/38.ts
#EXTINF:-1,Smash Rock Classics (Radio)
http://a3.smashmalta.com:82/live/smash/smash/37.ts
#EXTINF:-1,Smash Bella Italia (Radio)
http://a3.smashmalta.com:82/live/smash/smash/36.ts
#EXTINF:-1,Smash Arabija
http://a3.smashmalta.com:82/live/smash/smash/26.ts
#EXTINF:-1,Libya
http://a3.smashmalta.com:82/live/smash/smash/91.ts
#EXTINF:-1,TVM
http://a5.smashmalta.com:82/live/isuku/midoriya/7.ts
#EXTINF:-1,TVM2
http://a5.smashmalta.com:82/live/isuku/midoriya/6.ts
#EXTINF:-1,ONE
http://a5.smashmalta.com:82/live/isuku/midoriya/5.ts
#EXTINF:-1,NET
http://a5.smashmalta.com:82/live/isuku/midoriya/4.ts
#EXTINF:-1,Parlament ta' Malta TV
http://a5.smashmalta.com:82/live/isuku/midoriya/3.ts

**Reply by BellezaEmporium** [reply]
Xtream-Codes server.

**Reply by Sphinxroot** [reply]
look like but  comme from site  surely rebroacasting 

https://smashmalta.com/iptv%3A-pc%2Fmac

http://tv.smashmalta.com/  list from here

**Reply by BellezaEmporium** [reply]
> look like but comme from site surely rebroacasting
> 
> https://smashmalta.com/iptv%3A-pc%2Fmac
> 
> http://tv.smashmalta.com/ list from here

![image](https://user-images.githubusercontent.com/30985701/152313889-42ab91d9-bfc2-44ab-bd4c-4a55c17e445a.png)

**Comment by christopher3242** [comment]
I know you malta

**Comment by christopher3242** [comment]
Um Guys
